### PR TITLE
fix typos in tests and add library tests

### PR DESCRIPTION
**Pull request by archarbar**
Tests are now all working

### CODE REVIEW

File: test_elements.py [hunk 1]
```diff
@@ -22,25 +21,29 @@ def test_update_movie(self):
...
   
   def test_update_song(self):
     song = Song('Money Trees', 'Kendrick Lamar', 2012)
     song.update('Lose Yourself', 'Eminem', 2002)
-    self.assertEqual(serie.title, 'Lose Yourself')
-    self.assertEqual(serie.director, 'Eminem')
-    self.assertEqual(serie.year, 2002)
\ No newline at end of file
+    self.assertEqual(song.title, 'Lose Yourself')
+    self.assertEqual(song.artist, 'Eminem')
```
Comment (Tony9984) [inline test_elements.py:44]: should it be creator instead of artist here? since director was changed to creator above

File: test_library.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+from Library import *
+
+class TestStringMethods(unittest.TestCase):
+  def setUp(self):
+    self.library = Library([],[],[])
+
+  def test_new_library(self):
+    self.assertEqual(len(self.library.series), 0)
+    self.assertEqual(len(self.library.series), 0)
```
Comment (Tony9984) [inline test_library.py:11]: i think the same code is tested twice here

File: test_library.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    self.assertEqual(len(self.library.songs), 0)
+
+  def test_add_remove(self):
```
Comment (Tony9984) [inline test_library.py:14]: it might be better to test add and remove separately

File: test_library.py (hunk 2 above)
Comment (Tony9984) [reply test_library.py:11]: maybe it should be library.movies?

File: test_library.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  def test_song_exists(self):
+    song = Song('Lose Yourself', 'Eminem', 2002)
+    self.library.addSong(song)
+    self.assertTrue(self.library.doesSongExist('Lose Yourself'))
+
+  def test_song_not_exists(self):
+    self.assertFalse(self.library.doesSongExist('Lose Yourself'))
+
+if __name__ == '__main__':
+  unittest.main()
```
Comment (Tony9984) [inline test_library.py:110]: are there any dependencies to test? (i.e. relationships between movie, library, serie)

**Review by Tony9984 — CHANGES_REQUESTED** [review]
had a few questions about the test cases, otherwise looks good!

File: test_library.py (hunk 2 above)
Comment (archarbar) [reply test_library.py:11]: Good catch, that was a typo! It's now addressed

Comment (archarbar) [reply test_library.py:11]: Good catch, that was a typo! It's now addressed

File: test_library.py (hunk 3 above)
Comment (archarbar) [reply test_library.py:14]: They are tested separately in other tests, this is an integration test. I've moved it to its separated category for clarity

File: test_library.py (hunk 4 above)
Comment (archarbar) [reply test_library.py:110]: Nope, should be all good. Unit tests are handled in `test_elements.py`

**Review by Tony9984 — APPROVED** [review]
lgtm!

**Pull request merged**